### ISSUE TITLE
Install doesn't handle lack of root crontab

### ISSUE THREAD

**Issue by RogerMcCarty** [Matthias-Wandel/pi_blink_ip#5]
Runs great from a session, but when run on startup I simply can't decode the value. This is because the program wasn't installed in the non-existent crontab for root. No error was reported.

**Comment by RogerMcCarty** [comment]
My mistake. The problem concerned the fact that my root user did not have an existing crontab. So, the prog couldn't install and hence was not running. No wonder I couldn't decode the SD card accesses as an address.

**Comment by RogerMcCarty** [comment]
Any fix coming?

**Comment by Matthias-Wandel** [comment]
You said "your mistake"  I can't fix "your mistake".  Raspberry pi OS by default has a crontab for root.

**Comment by RogerMcCarty** [comment]
I'm very sorry that you believe that Raspberry Pi OS has a root crontab by default. It doesn't in the release I am running.

Robust, defensive, well-crafted software does not make assumptions. It's your software, do as you wish. I regret trying to help you with feedback. I expect you will want to have the last word on this.

**Comment by Matthias-Wandel** [comment]
If you can tell me what distribution doesn't have a root crontab, I can look into fixing this.  Wouldn't want to "fix" it without being able to test it.

**Comment by gijsvanoort** [comment]
I have installed the most recent full version of Raspberry Pi OS on a Raspberry Pi 4 ("Raspberry Pi OS Full, 32 bit, Released 2021-03-04" using Raspberry Pi Imager). In this distribution, root does _not_ hava a crontab; when doing `sudo crontab -l`, it replies 'no crontab for root' on stderr and returns exit code 1.

I created a fix for this (and tested it by playing around with no crontab (`sudo crontab -r`), crontab without pi_blink_ip.py in it and crontab with pi_blink_ip.py already installed. I'll create a pull request right now. Testing can be done by backing up the root cron tab, removing it and restoring the backup after test.
